src/sql/parts/profiler/contrib/profilerActions.contribution.ts

```diff
@@ -70,7 +70,6 @@ CommandsRegistry.registerCommand({
 		if (activeEditor instanceof ProfilerEditor) {
 			let profilerInput = activeEditor.input;
 			return profilerService.startSession(profilerInput.id).then(() => {
-				profilerInput.state.change({ isRunning: true, isStopped: false, isPaused: false });
 				return true;
```

Comment:
It looks like you can remove the `.then(() => { return true; });` part now since `profilerService.startSession()` already returns a `Thenable<boolean>`. Same for stop below

Reply:
Thanks for pointing that out, I fixed it in other spots but forgot about it here